sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuShuffleEnv.scala

```diff
@@ -27,11 +27,9 @@ class GpuShuffleEnv(rapidsConf: RapidsConf) extends Logging {
   private var shuffleCatalog: ShuffleBufferCatalog = _
   private var shuffleReceivedBufferCatalog: ShuffleReceivedBufferCatalog = _
 
-  private lazy val conf = SparkEnv.get.conf
-
   lazy val isRapidsShuffleConfigured: Boolean = {
-    conf.contains("spark.shuffle.manager") &&
-      conf.get("spark.shuffle.manager") == GpuShuffleEnv.RAPIDS_SHUFFLE_CLASS
+    rapidsConf.get("spark.shuffle.manager").isDefined &&
+      rapidsConf.get("spark.shuffle.manager").get == GpuShuffleEnv.RAPIDS_SHUFFLE_CLASS
```

Comment:
`RapidsConf` is not guaranteed to be a superset of `SparkConf`.  A SparkConf should be used to lookup a Spark config.

Reply:
The issue that sparkConf in this class is taken from SparkEnv, that's not initialized in case of command-line benchmark. The whole issue is that in order for Ucx to work GPUDeviceManager must be initialized. The only place that initializes `devicedId` is in [`initializeRmm`](https://github.com/NVIDIA/spark-rapids/blob/branch-0.6/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala#L254). In this method it initializes after GpuShuffleEnv:  https://github.com/NVIDIA/spark-rapids/blob/branch-0.6/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala#L270